style.css

```diff
@@ -550,6 +550,126 @@ label {
   align-items: center;
 }
 
+.popbackground {
+  background-color: #b3bac59c;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 100;
+  backdrop-filter: blur(6px);
+  position: fixed;
+}
+
+.popup-card {
+  position: fixed;
+  height: 95vh;
+  width: 90vw;
+  display: flex;
+  flex-direction: column;
+  z-index: 101;
+  background-color: #fff;
+  padding: 4.5%;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex-wrap: nowrap;
+}
```

Comment:
- [ ] Great job on working on the card for the mobile version but I think that here you forget to add the two buttons for live version and source code so kindly add them or show then by decrease the hight for the pop up a little bit 🙏🏻🙏🏻🔥

- Your Design
![image](https://user-images.githubusercontent.com/58553711/180114711-cbba08ef-2716-4ffd-9a6b-e4a61eb59824.png)

- Figma template
![image](https://user-images.githubusercontent.com/58553711/180114793-6b432bdb-75be-4b43-aad4-66ab247a73b6.png)